Automattic/wp-calypso
# Jetpack Backup: add some additional Tracks events on Staging Site settings

Related to https://github.com/Automattic/jetpack-backup-team/issues/489

## Proposed Changes

* Add `calypso_jetpack_backup_setup_staging_site_view` to track when a user sees the set-up staging site setting.
* Add `calypso_jetpack_backup_setup_staging_site_toggle` to track when a user clicks the toggle to set up a site as staging.
  * This will include a `toggle` prop: true (on) / false (off).

## Why are these changes being made?


* These specific Tracks events will help us validate the feature usage of the copy to staging feature.

## Testing Instructions



* Spin up a Jetpack Cloud live branch
* Open the Network tab in your developer tools
* Navigate to Jetpack Cloud > Settings
  * Once the `Set up as staging site` loads, you should see a request to `t.gif` with the `calypso_jetpack_backup_setup_staging_site_view` event.
* Try clicking the toggle to set up the site as staging
  * You should see a request to `t.gif` with the `calypso_jetpack_backup_setup_staging_site_toggle` event including a `toggle` prop.

## Pre-merge Checklist
